editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -113,6 +113,15 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 		} catch (CoreException ex) {
 			ProjectTemplatesActivator.pluginLog().logError(ex);
 		}
+		// delete invalid MANIFEST files
+		IResource rs = c.getProject().findMember("src/META-INF/");

```

Comment:
Why do we need to do it in the code? Shoudln't we just remove them from the zipped file?